guides/updating/versions/0.4-to-0.5.md

```diff
@@ -294,65 +294,15 @@ const data = 'openid-credential-offer://?credential_offer_uri=...'
 
 const resolvedCredentialOffer = await agent.modules.openId4VcHolder.resolveCredentialOffer(data)
 
-await agent.modules.openId4VcHolder.acceptCredentialOfferUsingPreAuthorizedCode(
-  // First parameter is now the resolved credential offer
+const token = await holder.modules.openId4VcHolderModule.requestToken({
   resolvedCredentialOffer,
-  {
-    // has been renamed from proofOfPossessionVerificationMethodResolver to credentialBindingResolver
-    credentialBindingResolver: async ({
-      supportedDidMethods,
-      keyType,
-      supportsAllDidMethods,
-      // supportsJwk now also passed
-      supportsJwk,
-      credentialFormat,
-    }) => {
-      // NOTE: example implementation. Adjust based on your needs
-      // Return the binding to the credential that should be used. Either did or jwk is supported
-
-      if (supportsAllDidMethods || supportedDidMethods?.includes('did:key')) {
-        const didResult = await agent.dids.create<KeyDidCreateOptions>({
-          method: 'key',
-          options: {
-            keyType,
-          },
-        })
-
-        if (didResult.didState.state !== 'finished') {
-          throw new Error('DID creation failed.')
-        }
-
-        const didKey = DidKey.fromDid(didResult.didState.did)
-
-        // you now need to return an object instead of VerificationMethod instance
-        // and method 'did' or 'jwk'
-        return {
-          method: 'did',
-          didUrl: `${didKey.did}#${didKey.key.fingerprint}`,
-        }
-      }
-
-      // we also support plain jwk for sd-jwt only
-      if (supportsJwk && credentialFormat === OpenId4VciCredentialFormatProfile.SdJwtVc) {
-        const key = await agent.wallet.createKey({
-          keyType,
-        })
-
-        // you now need to return an object instead of VerificationMethod instance
-        // and method 'did' or 'jwk'
-        return {
-          method: 'jwk',
-          jwk: getJwkFromKey(key),
-        }
-      }
+  userPin: '1234',
+});
 
-      throw new Error('Unable to create a key binding')
-    },
-
-    verifyCredentialStatus: false,
-    allowedProofOfPossessionSignatureAlgorithms: [JwaSignatureAlgorithm.EdDSA, JwaSignatureAlgorithm.ES256],
-  }
-)
+const credentials = await holder.modules.openId4VcHolderModule.requestCredentials({
+  resolvedCredentialOffer,
+  token,
```

Comment:
A lot of the parameters are removed which are still used. Also these methods were only introduced in later 0.5 versions so we might want to add this to 0.6 migration guide?